### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by akoniar**
Please @spisiakm review

### CODE REVIEW

File: various-db-engines-it/pom.xml [hunk 1]
```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
-      <version>42.1.4</version>
+      <version>42.2.2.jre7</version>
```
Comment (spisiakm) [inline various-db-engines-it/pom.xml]: Any reason to use `42.2.2.jre7` and not just `42.2.2`?

File: vertx-service-discovery-it/discovery-gateway/pom.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
-      <version>9.4.1212</version>
+      <version>42.2.2.jre7</version>
```
Comment (spisiakm) [inline vertx-service-discovery-it/discovery-gateway/pom.xml]: Same question as above.

File: pom.xml [hunk 3]
```diff
@@ -102,12 +102,12 @@
...
       <artifactId>agroal-api</artifactId>
-      <version>0.4.0.redhat-1</version>
+      <version>0.5</version>
     </dependency>
     <dependency>
       <groupId>io.agroal</groupId>
       <artifactId>agroal-pool</artifactId>
-      <version>0.4.0.redhat-1</version>
+      <version>0.5</version>
     </dependency>
```
Comment (spisiakm) [inline pom.xml]: Remove both agroal dependencies please. ENTVTX-170 should be resolved by now. Thanks! :)

File: various-db-engines-it/pom.xml (hunk 1 above)
Comment (akoniar) [reply various-db-engines-it/pom.xml]: Because I used $ mvn versions:use-latest-versions, and I didn't check that. Thanks! I will fix that.

**Comment by spisiakm** [comment]
Thanks @akoniar !

**Pull request merged**